pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/AndOperatorTransformFunction.java

```diff
@@ -45,4 +48,37 @@ int getLogicalFuncResult(int left, int right) {
     }
     return 0;
   }
+
+  @Nullable
+  @Override
+  public RoaringBitmap getNullBitmap(ValueBlock valueBlock) {
+    int numDocs = valueBlock.getNumDocs();
+    int numArguments = _arguments.size();
+    int[][] intValuesSVs = new int[numArguments][numDocs];
+    RoaringBitmap[] nullBitmaps = new RoaringBitmap[numArguments];
+    RoaringBitmap nullBitmap = new RoaringBitmap();
+    for (int i = 0; i < numArguments; i++) {
+      intValuesSVs[i] = _arguments.get(i).transformToIntValuesSV(valueBlock);
+      nullBitmaps[i] = _arguments.get(i).getNullBitmap(valueBlock);
+    }
+    for (int docId = 0; docId < numDocs; docId++) {
+      boolean isFalse = false;
+      for (int i = 0; i < numArguments; i++) {
+        if ((nullBitmaps[i] == null || !nullBitmaps[i].contains(docId)) && intValuesSVs[i][docId] == 0) {
```

Comment:
I don't understand this predicate. Specifically I don't understand why `!nullBitmaps[i].contains(docId)) && intValuesSVs[i][docId] == 0` implies that we don't need to check for the other docIds or arguments.

Reply:
For AND operator, if any of the argument is false, then the result is false.

Reply:
Oh! and the int 0 is the value we use as false. That makes sense. Maybe the reason is that it is early morning here and I'm not 100% woken up, but it looks confusing to me. I would prefer to see something like `BooleanUtils.isFalse(intValuesSVs[i])`, but that method requires an Object, so it would imply to do boxing.

We can create a `public static boolean isFalseInternalValue(int value)`, but I leave that decision to you.